packages/firestore/test/unit/specs/spec_test_runner.ts

```diff
@@ -231,7 +231,8 @@ abstract class TestRunner {
       TEST_PERSISTENCE_KEY,
       'host',
       /*ssl=*/ false,
-      /*forceLongPolling=*/ false
+      /*forceLongPolling=*/ false,
+      /*autoDetectLongPolling=*/ true
```

Comment:
I would use the default value here, since it matches the expected usage for most users.

Reply:
The constant I declared in `database.ts` isn't exported. Does that mean that single variable should be in order to reference it here?

Reply:
You can just hardcode "false" as we did for "forceLongPolling" above. Ideally, we should remove both settings pretty soon.